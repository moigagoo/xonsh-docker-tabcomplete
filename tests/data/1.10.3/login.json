{
        "arguments": [
                "-e", "--email",
                "--help",
                "-p", "--password",
                "-u", "--username"
        ],
        "commands": []
}
