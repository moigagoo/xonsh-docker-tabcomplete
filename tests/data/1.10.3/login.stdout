
Usage:	docker login [OPTIONS] [SERVER]

Register or log in to a Docker registry.
If no server is specified, the default is defined by the daemon.

  -e, --email        Email
  --help             Print usage
  -p, --password     Password
  -u, --username     Username
